### DISCUSSION TITLE
Config file read error

### DISCUSSION THREAD

**Discussion by yoongschoi** [nrc-cnrc/EGSnrc#1023] [Questions] [answered]
This error message pops up when I do 'egs_view slab.egsinp slab ptracks'. How can I fix this?

![config file error](https://github.com/nrc-cnrc/EGSnrc/assets/113466198/c412dac1-5ff0-45bc-9619-c880147c7190)

**Comment by rtownson** [comment, marked as answer]
You have a typo, you're just missing a ".", the command should be:
```
egs_view slab.egsinp slab.ptracks
```
Instead, the code thought you were providing 3 files, which meant the 3rd one was a config (.egsview) file.

**Reply by yoongschoi** [reply]
Sorted! Thank you so much
